corsheaders/middleware.py

```diff
@@ -148,6 +149,17 @@ def origin_found_in_white_lists(self, origin, url):
             (origin == 'null' and origin in conf.CORS_ORIGIN_WHITELIST) or
             self.regex_domain_match(origin)
         )
+    
+    def origin_found_in_cors_model_lists(self, origin, url):
+        if conf.CORS_MODEL is not None:
+            model = apps.get_model(*conf.CORS_MODEL.split('.'))
+            return (
+                model.objects.filter(cors=url.netloc).exists()
+            )
+        else:
+            return (
+                False
+            )
```

Comment:
this code would be better as `return False`. I would also drop the `else` and have the code at the same indentation level as the `if`